modules/core/04-channel/types/version_test.go

```diff
@@ -0,0 +1,43 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+func TestSplitVersions(t *testing.T) {
+	testCases := []struct {
+		name       string
+		version    string
+		mwVersion  string
+		appVersion string
+	}{
+		{
```

Comment:
could you add a test case for something like `fee29-1::ics20-1` and `:ics20-1`?

Reply:
This will not error or do anything here, it'll just pass on the malformed strings to the respective middleware/app which is responsible for erroring. I've tested that in the fee handshake tests